west.yml

```diff
@@ -22,16 +22,9 @@ manifest:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
 
-  group-filter: [-babblesim]
```

Comment:
wonder why this is needed :)

Reply:
I haven't been able to run west or twister without this revert